packages/react-components/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -176,6 +175,29 @@ export const useTooltip_unstable = (props: TooltipProps): TooltipState => {
     [setDelayTimeout, setVisible, state.showDelay, context],
   );
 
+  // Callback ref that attaches a keyborg:focusin event listener.
+  const keyborgListenerCallbackRef = React.useMemo(() => {
```

Comment:
```suggestion
  const keyborgListenerCallbackRef = React.useCallback(() => {
```

The [React docs for useMemo](https://react.dev/reference/react/useMemo#usememo) state

> You should only rely on useMemo as a performance optimization. If your code doesn’t work without it, find the underlying problem and fix it first. Then you may add useMemo to improve performance.

Reply:
This can't be replaced directly by `useCallback` since it's storing state (`current`) in the init function's closure. I've switched it to use `useState`, which has lifetime guarantees by React.